src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/PackagePropertyPage/AssemblyVersionValueProvider.cs

```diff
@@ -22,7 +22,7 @@ internal sealed class AssemblyVersionValueProvider : BaseVersionValueProvider
                 return s_DefaultAssemblyVersion;
             }
 
-            return new Version(defaultVersion.Major, defaultVersion.Minor, defaultVersion.Build, revision: 0);
+            return new Version(defaultVersion.Major, defaultVersion.Minor, Math.Max(defaultVersion.Build, 0), revision: Math.Max(defaultVersion.Revision, 0));
```

Comment:
See https://github.com/dotnet/sdk/issues/574. Build computes it using a target, which is not executed during Evaluation. Once we fix the SDK issue, we can remove the interceptor in project system - https://github.com/dotnet/roslyn-project-system/issues/1017